moto/ses/responses.py

```diff
@@ -507,6 +524,57 @@ def describe_receipt_rule(self):
   </ResponseMetadata>
 </CreateReceiptRuleResponse>"""
 
+DESCRIBE_RECEIPT_RULE_SET = """<DescribeReceiptRuleSetResponse xmlns="http://ses.amazonaws.com/doc/2010-12-01/">
+  <DescribeReceiptRuleSetResult>
```

Comment:
I got the shape of the XML response by inferring it from [here](https://github.com/boto/botocore/blob/develop/botocore/data/ses/2010-12-01/service-2.json#L2032) and the [SES docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ses.html#SES.Client.describe_receipt_rule_set)

@bblommers I recall there being a suggestion to set up a MITM proxy to see the XML response directly, unfortunately, I don't have experience with this and I'm not sure where to start - could you point me in the right direction? That being said, I'm pretty confident that this produces the expected response.

Reply:
Well, the tests pass, so boto3 seems to think it's correct - that's the most important part!

For proxying, I've always used [MITMProxy](https://mitmproxy.org/). 
On Linux (/Ubuntu) the installation/setup is as easy as:
```
sudo apt install mitmproxy
mitmweb
```
YMMV of course - it will depend on what OS you're using. 
MITMWeb will start a browser window where you can easily see the requests that have gone through the proxy.

After that, the easiest way of proxying requests to AWS is probably via the CLI.
```
export HTTP_PROXY=http://localhost:8080
export HTTPS_PROXY=http://localhost:8080
aws ses describe-rule-set --no-verify-ssl  # Do not verify SSL requests, as the proxy does not have SSL certificates set-up
```
It is possible to use a proxy using boto3 as well - as long as both the proxy is configured, and boto3 is configured to not verify SSL certificates.

Note that this only works for requests to AWS itself - Moto will intercept requests before they are send to the proxy.